reconcile/external_resources/model.py

```diff
@@ -181,14 +204,21 @@ def image_version(self) -> str:
     def resolve_configuration(
         module: ExternalResourcesModuleV1, spec: ExternalResourceSpec
     ) -> "ExternalResourceModuleConfiguration":
-        # TODO: Modify resource schemas to include this attributes
-        data = {
-            "image": module.image,
-            "version": module.default_version,
-            "reconcile_drift_interval_minutes": module.reconcile_drift_interval_minutes,
-            "reconcile_timeout_minutes": module.reconcile_timeout_minutes,
-        }
-        return ExternalResourceModuleConfiguration.parse_obj(data)
+        if not (module_overrides := spec.metadata.get("module_overrides")):
+            module_overrides = ExternalResourcesModuleOverridesV1(
```

Comment:
seems we always want to assign `module_overrides`, then can simplify it

```python
module_overrides = spec.metadata.get("module_overrides") or ExternalResourcesModuleOverridesV1(...)
```

Reply:
fixed.